File: persoon.cs
using System;

public class Persoon {
    public string naam;
    private int leeftijd;
    protected char geslacht;
    public bool isStudent;
    public double gemiddeldCijfer;

    public Persoon(string naam, int leeftijd, char geslacht, bool isStudent, double gemiddeldCijfer) {
        this.naam = naam;
        this.leeftijd = leeftijd;
        this.geslacht = geslacht;
        this.isStudent = isStudent;
        this.gemiddeldCijfer = gemiddeldCijfer;
    }

    public int Leeftijd {
        get { return leeftijd; }
        set { leeftijd = value; }
    }
}

File: persoon.py
class Persoon:
    def __init__(self, naam, leeftijd, geslacht, isStudent, gemiddeldCijfer):
        self.naam = naam
        self._leeftijd = leeftijd
        self._geslacht = geslacht
        self.isStudent = isStudent
        self.gemiddeldCijfer = gemiddeldCijfer

    @property
    def leeftijd(self):
        return self._leeftijd

    @leeftijd.setter
    def leeftijd(self, value):
        self._leeftijd = value

    @property
    def geslacht(self):
        return self._geslacht

    @geslacht.setter
    def geslacht(self, value):
        self._geslacht = value
